src/_utils/radio/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+
+import { color } from '_utils/branding'
+import Loader from 'loader'
+import CircleIcon from 'icon/circleIcon'
+
+interface RadioProps {
+  readonly isChecked?: boolean
+  readonly isLoading?: boolean
+}
+
+const Radio = ({ isChecked = false, isLoading = false }: RadioProps) => {
```

Comment:
I think we should rename this, see my other comment in the snapshot

Reply:
Renamed ;)